Simplenote/build.gradle

```diff
@@ -103,6 +103,13 @@ android {
         }
     }
 
+    packagingOptions {
+        exclude 'META-INF/LICENSE-LGPL-3.txt'
+        exclude 'META-INF/DEPENDENCIES'
+        exclude 'META-INF/LICENSE-LGPL-2.1.txt'
+        exclude 'META-INF/LICENSE-W3C-TEST'
```

Comment:
There seemed to be a conflict with Flexmark and another library with these files. Adding these exclusions fixed compiling for me.